docs/security.md

```diff
@@ -207,6 +207,15 @@ The following table describes the different options available for configuring th
   </td>
   <td>2.2.0</td>
 </tr>
+<tr>
+  <td><code>spark.network.crypto.cipher</code></td>
+  <td>AES/CTR/NoPadding</td>
+  <td>
+    Cipher mode to use. Defaults "AES/CTR/NoPadding" for backward compatibility, which is not authenticated. 
+    Recommended to use "AES/GCM/NoPadding", which is an authenticated encryption mode.
+  </td>
+  <td>4.0.0</td>
```

Comment:
@mridulm  Can we still add the new feature since 4.0.0-preview1 has been pass?

Reply:
That was a preview release, I am sure we will have additional new features landing and existing behavior evolving before 4.0 RC gets cut :-)

Note that this specific PR will also get backported to 3.5 and 3.4 - given the security concerns it is mitigating.
So would love to hear your thoughts @Ngone51 - particularly with the intent to derisk the change.

If you need more context on why we need to backport this to 3.5/3.4, @sweisdb could provide you more context via email.